### PR TITLE
Leaderboard milestone 3

### PR DESCRIPTION
**Pull request by Anny85-code**
### In this milestone I;
- [x] Added  Styling to the project 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # leaderboard
 
 > "leaderboard" is a simple web application that displays a table of names and scores and allows you to add and remove names and scores from that table.
-> ![leaderboard](https://user-images.githubusercontent.com/87186552/151043785-59ac7cb6-5545-4120-9b6f-1183399d1a50.png)
+> ![leaderboardlatest](https://user-images.githubusercontent.com/87186552/151335363-85db4e6d-d5bf-4449-aae8-d132792f1adc.png)
```
Comment (mikethreels) [inline README.md:4]: - [ ] in the License part of the readme file you added the MIT license which is great, but the link to the actual license is not working, please have a look at [the Microverse template](https://github.com/microverseinc/readme-template) to see what it should look like.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Anny, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Anny85-code) [reply README.md:4]: Thank you for your review, the changes have been applied.

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Anny85-code, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**